install.sh

```diff
@@ -13,10 +71,15 @@ if [ ! -d "$DVM_DIR" ]; then
 fi
 
 echo "Downloading dvm.sh..."
-curl -s -o $DVM_DIR/dvm.sh https://raw.githubusercontent.com/getcarina/dvm/master/dvm.sh
+dvm_download -L -C - --progress-bar https://raw.githubusercontent.com/getcarina/dvm/master/dvm.sh -o $DVM_DIR/dvm.sh
+
+if [ ! -f "${DVM_DIR}/dvm-helper" ]; then
```

Comment:
We are unconditionally overwriting dvm.sh but not dvm-helper. Can we remove the if check for dvm-helper and just always ensure that they have the latest when running the install?